jwala-services/src/test/java/com/cerner/jwala/service/group/impl/GroupJvmControlServiceImplTest.java

```diff
@@ -48,4 +51,12 @@ public void testControlGroupWithInvalidGroup() {
         ControlGroupJvmRequest controlGroupJvmRequest = new ControlGroupJvmRequest(groupId, JvmControlOperation.START);
         cut.controlGroup(controlGroupJvmRequest, testUser);
     }
+
+    @Test
+    public void testcheckSameState(){
```

Comment:
Add tests for the other states that return true for isStarted as well:
Initializing, initialized, start, starting, stop, stop, stopping, destroying